examples/python/face_tracking/face_tracking.py

```diff
@@ -20,6 +20,10 @@
 import tqdm
 from mediapipe.tasks.python import vision
 
+# If set, log everything as static
```

Comment:
```suggestion
# If set, log everything as static.
#
# Generally, the Viewer accumulates data until its set memory budget at which point it will
# remove the oldest data from the recording (see https://rerun.io/docs/howto/limit-ram)
# By instead logging data as static, no data will be accumulated over time since previous
# data is overwritten.
# Naturally, the drawback of this is that there's no history of previous data sent to the viewer,
# as well as no timestamps, making the Viewer's  timeline effectively inactive.
```

Reply:
nice